webapp/src/components/attach_comment_to_issue_menu/index.tsx

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {MouseEvent, useCallback} from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch, useSelector } from 'react-redux';
+import { GlobalState } from 'mattermost-redux/types/store';
+import {getPost} from 'mattermost-redux/selectors/entities/posts';
+import {isSystemMessage} from 'mattermost-redux/utils/post_utils';
+
+import {id as pluginId} from 'src/manifest';
+import GitLabIcon from 'src/images/icons/gitlab';
+import {openAttachCommentToIssueModal} from 'src/actions';
+
+interface PropTypes {
+    postId: string;
+};
+
+interface states {
+    connected: boolean
+}
+
+interface CurrentState extends GlobalState {
+    plugin: states;
+}
+
+const AttachCommentToIssuePostMenuAction = ({postId}: PropTypes) => {
+    const {show} = useSelector((state: CurrentState) => {
+        const post = getPost(state, postId);
+        const systemMessage = post ? isSystemMessage(post) : true;
```

Comment:
I think I suggested to change this in another PR. Do that change here and everywhere.